lib/ffi/radix_tree.rb

```diff
@@ -135,6 +139,20 @@ def get(key)
         ::FFI::RadixTree.match_free(byte_pointer) if byte_pointer
       end
 
+      def set(key, value)
```

Comment:
we already have `push` and now `set`; I think we should name it `update` or `swap` or `replace` ... just something that communicates the action better

Reply:
I was trying to make a `set` to go along with the `get`.   `get` only exists on the Ruby side, it calls `fetch` on the c++ side.  So my `set` function in Ruby calls `update` in c++.  But I can change them to both be `update` if you'd like.